redirect.js

```diff
@@ -0,0 +1,16 @@
+// GENERATED AUTOMATICALLY, DO NOT EDIT
+let list = "";
+
+list = JSON.parse(window.atob(list));
+
+// normalize path. lower case, remove leading slashes.
+const from = window.location.pathname.toLowerCase().replace(/^(\/+)/, "");
+
+// find corresponding "to" field
+const to = list.find((entry) => entry?.from === from)?.to;
+
+// enable "preserve log" in browser dev tools to still see this after redirect
+console.info({ list, from, to });
+
+// navigate to destination
+if (to) window.location.href = to;
```

Comment:
Does this have a potential to be blocked by a client due to built-in security checks? Would [`location.assign()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location#example_1_navigate_to_a_new_page) offer any benefits?

Reply:
This is the typical way to navigate for as long as I can remember. I can't find any evidence that there's a difference between location.assign/replace and location.href = "". Did you see something?

Reply:
I'm less familiar with how this might operate within various clients so would defer to your best judgement here.